tests/e2e/e2e_meshrootcertificate_test.go

```diff
@@ -49,38 +45,106 @@ var _ = OSMDescribe("MeshRootCertificate",
 		})
 	})
 
+// basicCerRotationScenario rotates a new cert in through the follow pattern:
+// | Step  | MRC1 old   | MRC2 new     | signer    | validator |
+// | ----- | ---------- | ------------ | --------- | --------- |
+// | 1     | active     |              | mcr1      | mcr1      |
```

Comment:
```suggestion
// | 1     | active     |              | mrc1      | mrc1      |
```

Reply:
I don't see a difference in the suggested change. Let me know if I am missing something!

Reply:
m_**c**_r -> m_**r**_c

Reply:
Got it! Thank you!